### ISSUE TITLE
[BOUNTY] Veymed Limbs

### ISSUE THREAD

**Issue by ARandomHyena** [SPLURT-Station/S.P.L.U.R.T-Station-13#296]
## About the Bounty
5 Dollar bounty to port Veymed limbs from Citadel, look identical to normal non-prosthetic limbs but are healed like prosthetics. The Prosthetics look ugly and I want a better looking alternate.

**Comment by Zirok-BYOND** [comment]
Are you still interested in this?

**Comment by ARandomHyena** [comment]
Yes, I am. Never thought anyone would take it up, but, I am interested.
